charts/nutanix-csi-storage/values.yaml

```diff
@@ -132,30 +147,59 @@ servicemonitor:
 
 controller:
   replicas: 2
-  image: quay.io/karbon/ntnx-csi:v2.6.6
+  image: quay.io/karbon/ntnx-csi:v3.0.0-beta1
   nodeSelector: {}
   tolerations: []
+  cpu: 200m
+  memory: 200Mi
 
 node:
-  image: quay.io/karbon/ntnx-csi:v2.6.6
-  nodeSelector: {}
+  image: quay.io/karbon/ntnx-csi:v3.0.0-beta1
+  nodeSelector:
+    kubernetes.io/os: linux
   tolerations: []
+  cpu: 200m
+  memory: 200Mi
 
 sidecars:
   registrar:
     image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.9.1
   provisioner:
     image: registry.k8s.io/sig-storage/csi-provisioner:v3.6.2
     imageLegacy: registry.k8s.io/sig-storage/csi-provisioner:v2.2.2
+  attacher:
+    image: k8s.gcr.io/sig-storage/csi-attacher:v3.4.0
```

Comment:
@subodh01 this attacher version is no more supported
expected version is v4.4.2

ok to upgrade

Reply:
Yes.